packages/electron-app/src/main/util/csp.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { IS_PROD } from './constants';
+
+/* eslint-disable */
+// References:
+// * https://github.com/parity-js/shell
+// * https://github.com/paritytech/fether
+const CSP_CONFIG = {
+  // Disallow mixed content
+  blockAllMixedContent: 'block-all-mixed-content;',
+  // Disallow framing and web workers.
+  // tslint:disable-next-line:quotemark
+  childSrc: "child-src 'none';",
+  // FIXME - Only allow connecting to WSS and HTTPS servers.
+  connectSrc: 'connect-src http: ws:;',
+  // Fallback for missing directives.
+  // Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src
+  //
+  // Disallow everything as fallback by default for all CSP fetch directives.
+  defaultSrc: "default-src 'none';",
+  // Disallow fonts.
+  fontSrc: "font-src 'none';", // Additionally used in Parity-JS Shell `'self' data: https:`
```

Comment:
```
Refused to load the font 'http://localhost:3000/static/media/icons.0ab54153.woff2' because it violates the following Content Security Policy directive: "font-src 'none'".
```

I guess we can put font-src 'self'